roles/kubernetes-apps/metallb/defaults/main.yml

```diff
@@ -5,3 +5,11 @@ metallb_protocol: "layer2"
 metallb_port: "7472"
 metallb_limits_cpu: "100m"
 metallb_limits_mem: "100Mi"
+# metallb_protocol: "bgp"
+# metallb_peers:
+#   - peer_address: 192.0.2.1
+#     peer_asn: 64512
+#     my_asn: 4200000000
+#   - peer_address: 192.0.2.2
+#     peer_asn: 64513
+#     my_asn: 4200000000
```

Comment:
I feel we don't need to put comment-ed non-default values on `defaults/main.yml` as the file name.
We might want to put these comment-ed values to understand what are options easily, and ` inventory/sample/group_vars/k8s-cluster/addons.yml` shows that.